### PR TITLE
feat(path-to-regexp): path-to-regexp 8.1.0 update

### PR DESCRIPTION
**Pull request by frankkeefer**
* Change path-to-regexp 6.2.2 to 8.1.0
* Update code to handled changes in path-to-regexp v6.x.x -> v8.x.x

### CODE REVIEW

File: src/middlewares/openapi.metadata.ts [hunk 1]
```diff
@@ -81,14 +81,16 @@ export function applyOpenApiMetadata(
...
-      const keys = [];
+      let keys = [];
       const strict = !!req.app.enabled('strict routing');
       const sensitive = !!req.app.enabled('case sensitive routing');
       const pathOpts = {
         sensitive,
         strict,
       };
-      const regexp = pathToRegexp(expressRoute, keys, pathOpts);
+      const regexpObj = pathToRegexp(expressRoute, pathOpts);
```
Comment (cvchauhan) [inline src/middlewares/openapi.metadata.ts:91]: we can remove let keys = [];
and directly use 
 const { regexpObj, keys } = pathToRegexp(expressRoute, pathOpts);

**Review by cvchauhan — CHANGES_REQUESTED** [review]
we can remove let keys = []; and keys = regexpObj.keys;
and directly use
const { regexpObj, keys } = pathToRegexp(expressRoute, pathOpts);

File: src/middlewares/openapi.metadata.ts (hunk 1 above)
Comment (frankkeefer) [reply src/middlewares/openapi.metadata.ts:89]: Added in requested changes. 
* removed let keys =[] and related uses
* removed const regexp = regepObj.regexp; and related uses
All current unit tests still pass locally

**Review by cvchauhan — APPROVED** [review]

**Review by cdimascio — APPROVED** [review]
thank you. this is great

**Comment by dogmatic69** [comment]
Please create a release 🙏🏻

**Pull request merged**